src/ducks/SANCharts/metrics/data.js

```diff
@@ -568,6 +573,13 @@ export const tooltipSettings = {
   }
 }
 
+SOCIAL_TWITTER_INTERVALS.forEach(interval => {
+  tooltipSettings['twitter_followers_' + interval] = {
+    label: 'Twitter changes',
```

Comment:
Both metrics (`24h`, `7d`) will have the same label on the tooltip - `"Twitter changes"`, this might be confusing to the user
![image](https://user-images.githubusercontent.com/25135650/77067266-bbfc4e80-69f5-11ea-8e6b-495e07cc4332.png)